simpleSequential/test_integertoString.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from integerToString import integerToString
+
+class TestIntegerToString(unittest.TestCase):
+    def test_NegativeIntegerInput(self):
+        self.assertEqual(integerToString(-5), "-5")
+
+    def test_PositiveIntegerInput(self):
```

Comment:
please fix the typo, we are not capitalizing the first letter of the test definition.